src/Console/ServerBlock.php

```diff
@@ -87,17 +91,77 @@ protected function doExecute()
 				return $this->addBlockedServer($this->config);
 			case 'remove':
 				return $this->removeBlockedServer($this->config);
+			case 'export':
+				return $this->exportBlockedServers($this->config);
+			case 'import':
+				return $this->importBlockedServers($this->config);
 			default:
 				throw new CommandArgsException('Unknown command.');
 				break;
 		}
 	}
 
 	/**
-	 * Prints the whole list of blocked domains including the reason
+	 * Exports the list of blocked domains including the reason for the
+	 * block to a CSV file.
+	 *
+	 * @param IConfig $config
+	 */
+	private function exportBlockedServers(IConfig $config)
+	{
+		$filename = $this->getArgument(1);
+		$blocklist = $config->get('system', 'blocklist', []);
+		$fp = fopen($filename, 'w');
+		foreach ($blocklist as $domain) {
+			fputcsv($fp, $domain);
+		}
+	}
+	/**
+	 * Imports a list of domains and a reason for the block from a CSV
+	 * file, e.g. created with the export function.
 	 *
 	 * @param IConfig $config
 	 */
+	private function importBlockedServers(IConfig $config)
+	{
+		$filename = $this->getArgument(1);
+		$currBlockList = $config->get('system', 'blocklist', []);
+		$newBlockList = [];
+		if (($fp = fopen($filename, 'r')) !== FALSE) {
```

Comment:
Please add an error message if the file can't be open. Additionally, we use lowercase `false`.
```suggestion
		if (($fp = fopen($filename, 'r')) !== false) {
```